<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QDialog" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>привет я график</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>501</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buildButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>build graph</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>174</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="testlabel">
      <property name="text">
       <string>сейчас будем строить график функции:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="expression_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>укажите область определения графика:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>x_begin = </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_begin">
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>-10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>x_end = </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_end">
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>y_begin = </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="y_begin">
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>-10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>y_end = </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="y_end">
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>500</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Status bar:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>libs/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
